spec/features/show_page_buttons_spec.rb

```diff
@@ -0,0 +1,46 @@
+# Generated via
+#  `rails generate hyrax:work Pdf`
```

Comment:
Was this code built by this generator?

Reply:
Ha! In a sense... I'll remove this comment